lib/log.bash

```diff
@@ -4,45 +4,56 @@ export BASH_IT_LOG_LEVEL_ERROR=1
 export BASH_IT_LOG_LEVEL_WARNING=2
 export BASH_IT_LOG_LEVEL_ALL=3
 
+function _has_colors()
+{
+    # Check that stdout is a terminal
```

Comment:
The indentation in this function is wrong (4 spaces instead of 2). Seems to have happened during copy/paste from the `appearance` lib file.

Reply:
done, nice catch!